### DISCUSSION TITLE
Why do most programmers have anime girls and cats as their pfps?

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#63200] [Other Feature Feedback, Questions, & Ideas] [answered]
I still don't understand

**Comment by fedorkanan** [comment, marked as answer]
because there weren't growing up since theirs childhood and wanna look cool from their unrealistic PFPs.

**Reply by bn-l** [reply]
but do Anime girls make you look cool?

**Comment by Hyadawild** [comment]
from my point of view as a weebs it increases self-confidence not only that anime profile pictures also increase quality because there is a waifu / husbu damage bonus

**Reply by bn-l** [reply]
Good explanation thanks. Also, kawaii wifu.
